daemon/cmd/hubble.go

```diff
@@ -50,28 +50,36 @@ func (d *Daemon) launchHubble() {
 		logger.WithError(err).Error("Failed to initialize Hubble")
 		return
 	}
-	s, err := hubbleServer.NewLocalServer(payloadParser, logger,
+	observerServer, err := hubbleServer.NewLocalServer(payloadParser, logger,
 		serveroption.WithMaxFlows(option.Config.HubbleFlowBufferSize),
 		serveroption.WithMonitorBuffer(option.Config.HubbleEventQueueSize),
 		serveroption.WithCiliumDaemon(d))
 	if err != nil {
 		logger.WithError(err).Error("Failed to initialize Hubble")
 		return
 	}
-	go s.Start()
-	d.monitorAgent.GetMonitor().RegisterNewListener(d.ctx, hubble.NewHubbleListener(s))
+	go observerServer.Start()
+	d.monitorAgent.GetMonitor().RegisterNewListener(d.ctx, hubble.NewHubbleListener(observerServer))
 
-	listeners, err := hubbleServe.SetupListeners(addresses, api.CiliumGroupName)
+	srv, err := hubbleServe.NewServer(logger,
+		hubbleServe.WithListeners(addresses, api.CiliumGroupName),
```

Comment:
I think instead of adding the unix domain socket to the `addresses` in 41, we could now use the `WithUnixSocketListener` for `option.Config.HubbleSocketPath` here.

Reply:
This could be done but I didn't do it because I'd have to modify how logging and this code will anyway be changed again when the new peers service is enabled (needs a PR in hubble, then in cilium but it's almost ready). There will indeed be 2 hubble servers, one listening on the unix domain socket which will include the peers gRPC service and one for all other addresses which will not.